cmd/cstor-pool-mgmt/controller/pool-controller/after_queue_handler.go

```diff
@@ -0,0 +1,192 @@
+package poolcontroller
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/controller/common"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/pool"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/util"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+)
+
+// syncHandler compares the actual state with the desired, and attempts to
+// converge the two. It then updates the Status block of the cStorPoolUpdated resource
+// with the current status of the resource.
+func (c *CStorPoolController) syncHandler(key, operation string) error {
+	glog.Infof("at sync handler")
```

Comment:
Add the key and operation values to the log message. If this is too redundant, consider making it debug type, else these messages may fill up the entire log file !!

Reply:
will remove this. Not required.